### PR TITLE
Add recipe for harminv

### PR DESCRIPTION
**Pull request by ChristopherHogan**

### CODE REVIEW

**Comment by ChristopherHogan** [comment]
This has been green with no attention for 13 days.
@conda-forge/staged-recipes

**Comment by CJ-Wright** [comment]
This looks like it use some review from @conda-forge/help-c-cpp

**Comment by scopatz** [comment]
Hi @ChristopherHogan - whenever the PR is ready for review, please ping @conda-forge/core (or a related team).  Otherwise we will miss it. Thanks!

**Comment by scopatz** [comment]
(ready for review = CI passes or you need help or your have a question related to that specific package.)

File: recipes/harminv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - gcc
+    - libgfortran  # [linux]
+    - openblas
+  run:
+    - libopenblas
+    - libgfortran
+
+test:
+  source_files:
+    - sines
```
Comment (scopatz) [inline recipes/harminv/meta.yaml]: This doesn't seem to have a real test.  At the very least, could you add an existence test please https://conda-forge.org/docs/testing.html#simple-existence-tests

File: recipes/harminv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - libgfortran  # [linux]
+    - openblas
+  run:
+    - libopenblas
```
Comment (isuruf) [inline recipes/harminv/meta.yaml]: Use `openblas` here.

File: recipes/harminv/build.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix="${PREFIX}" --enable-shared --with-blas=openblas
+make
+make install
```
Comment (scopatz) [inline recipes/harminv/build.sh]: Can you run `make check`?

File: recipes/harminv/meta.yaml (hunk 2 above)
Comment (ChristopherHogan) [reply recipes/harminv/meta.yaml]: Fixed.

File: recipes/harminv/build.sh (hunk 3 above)
Comment (ChristopherHogan) [reply recipes/harminv/build.sh]: `make check` runs the same test I was running in `meta.yaml`, so I changed `meta.yaml` to just do the existence tests, and now I'm running `make check`.

**Comment by ChristopherHogan** [comment]
This has addressed all feedback and is passing all tests.

**Pull request merged**